evm/src/executor/abi.rs

```diff
@@ -17,6 +17,7 @@ ethers::contract::abigen!(
             roll(uint256)
             warp(uint256)
             fee(uint256)
+            bank(address)
```

Comment:
I think naming the cheatcode `coinbase` would be more clear/intuitive, and consistent with `fee` and `chainId`. Otherwise LGTM! 🙂

Reply:
Sounds good, updated accordingly